api/src/test/java/marquez/TagIntegrationTest.java

```diff
@@ -29,6 +30,29 @@ public void testApp_testTags() {
     assertThat(taggedDataset.getTags()).contains("TESTDATASETTAG");
   }
 
+  @Test
+  public void testApp_testCreateTag() {
+    Tag tag = client.createTag("tag", "Description");
+    assertThat(tag.getName()).isEqualTo("tag");
+    assertThat(tag.getDescription()).contains("Description");
+
+    Tag tagWithoutDescription = client.createTag("tag2");
+    assertThat(tagWithoutDescription.getName()).isEqualTo("tag2");
+    assertThat(tagWithoutDescription.getDescription()).isEmpty();
+  }
+
+  @Test
+  public void testApp_testUpsertTag() {
+    Tag tag = client.createTag("tag", "Description");
+    assertThat(tag.getName()).isEqualTo("tag");
+    assertThat(tag.getDescription()).contains("Description");
+
+    Tag tagWithoutDescription = client.createTag("tag", "New Description");
```

Comment:
Maybe I found a bug in TagDao. Is it correct behaviour that I can't change the description after the initial creation of the tag.

Reply:
Nope, it definitely should! Mind opening an issue? And feel free to assign it to yourself if you wanted to take it on :)